### PR TITLE
nimbus.mainnet: enable full light client data

### PR DESCRIPTION
**Pull request by zah**

### CODE REVIEW

File: ansible/group_vars/nimbus.mainnet.yml [hunk 1]
```diff
@@ -31,7 +31,9 @@ beacon_node_subscribe_all: true
 # HTTP RPC support is unstable
 beacon_node_web3_urls: '{{ beacon_node_web3_urls_all | reject("match", "^http://") }}'
 # Light client data
-beacon_node_light_client_data_enabled: '{{ node.branch in ["unstable", "libp2p"] }}'
+beacon_node_light_client_data_enabled: true
+beacon_node_light_client_data_serve: true
+beacon_node_light_client_data_import: 'full'
```
Comment (etan-status) [inline ansible/group_vars/nimbus.mainnet.yml:36]:
```suggestion
beacon_node_light_client_data_import_mode: 'full'
```

Keep in mind, the actual launch arguments have been renamed on `unstable`, so may want to wait until the renames are also on `stable`, should be in 22.05 release

Comment (etan-status) [reply ansible/group_vars/nimbus.mainnet.yml:36]: https://github.com/status-im/nimbus-eth2/pull/3740

Comment (etan-status) [reply ansible/group_vars/nimbus.mainnet.yml:36]: Note: `beacon_node_light_client_data_serve` is now `true` by default: https://github.com/status-im/nimbus-eth2/pull/3921

Comment (etan-status) [reply ansible/group_vars/nimbus.mainnet.yml:36]: Note: `beacon_node_light_client_data_serve` is now `true` by default: https://github.com/status-im/nimbus-eth2/pull/3550

`beacon_node_light_client_data_enabled` is still needed (this toggles whether the LC data specific CLI options are passed to the beacon node; without this, the overrides don't have any effect)

**Comment by etan-status** [comment]
Is it possible to apply the setting only to API nodes? Setting the LC data import mode to `full` has the side effect of having slower startup times, so it should be avoided on servers that also run validators where uptime is more critical.

There's also a seemingly unrelated change to `beacon_node_web3_urls`. Intended?

**Comment by etan-status** [comment]
Actually, this would be cool on Goerli API nodes as well. There will be a REST interface in the future (~1mo), and having our public API nodes serve full historic LC data would be useful.

**Comment by jakubgs** [comment]
Rebase artifact.

File: ansible/group_vars/nimbus.mainnet.yml [hunk 2]
```diff
@@ -60,7 +60,10 @@ beacon_node_validator_monitor_totals: '{{ (node.public_api is defined and node.p
 beacon_node_dist_validators_enabled: false
 # Bootnodes should subscribe to all subnets
 beacon_node_subscribe_all: true
-# HTTP RPC support is unstable
+# Light client data
+beacon_node_light_client_data_enabled: true
```
Comment (etan-status) [inline ansible/group_vars/nimbus.mainnet.yml:64]:
```suggestion
beacon_node_light_client_data_enabled: '{{ (node.public_api is defined and node.public_api) }}'
```

**Review by etan-status — APPROVED** [review]

**Comment by jakubgs** [comment]
Deployed:
```
metal-01.he-eu-hel1.nimbus.mainnet | CHANGED | rc=0 >>
# /etc/systemd/system/beacon-node-mainnet-unstable-01.service
    --light-client-data-serve=true \
    --light-client-data-import-mode=full \
metal-02.he-eu-hel1.nimbus.mainnet | CHANGED | rc=0 >>
# /etc/systemd/system/beacon-node-mainnet-testing-01.service
    --light-client-data-serve=true \
    --light-client-data-import-mode=full \
```

**Comment by jakubgs** [comment]
I have deployed the same thing to Prater: https://github.com/status-im/infra-nimbus/commit/759b3c8b
```
linux-01.he-eu-hel1.nimbus.prater | CHANGED | rc=0 >>
# /etc/systemd/system/beacon-node-prater-unstable.service
    --light-client-data-serve=true \
    --light-client-data-import-mode=full \
linux-02.he-eu-hel1.nimbus.prater | CHANGED | rc=0 >>
# /etc/systemd/system/beacon-node-prater-testing.service
    --light-client-data-serve=true \
    --light-client-data-import-mode=full \
```

**Comment by etan-status** [comment]
Thanks! Depending on when those nodes were originally synced, it may take a while to re-index, making the first startup quite slow. That's a one-time indexing operation though.

**Pull request merged**